torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -849,5 +847,17 @@ std::vector<std::string> export_opnames(const script::Module& m) {
   return std::vector<std::string>(names.begin(), names.end());
 }
 
+// Thread local flag (only happens in export, i.e. on server side)
+// to control if instructions for bytecode default inputs are emitted
+// or not. It's the major difference between bytecode v5 and v6.
+thread_local bool emitBytecodeDefaultInputs =
+    caffe2::serialize::kProducedBytecodeVersion <= 5 ? true : false;
```

Comment:
Why not just set this to false?
If the only intention for the condition is to signal this is new logic for v6 then the comment should suffice.

Otherwise I think it looks odd to have a condition based on a constant.

Reply:
Makes sense. I'll set it false.